code/modules/power/singularity/singularity.dm

```diff
@@ -195,6 +202,9 @@
 				dissipate_delay = 5
 				dissipate_track = 0
 				dissipate_strength = 5
+				if(!warp)
+					warp = new(src)
+					vis_contents += warp
```

Comment:
is there a case where the singularity may immediately jump from stage one to stage three in a single go?